### ISSUE TITLE
Read time control in a structured way

### ISSUE THREAD

**Issue by mliebelt** [mliebelt/pgn-parser#8]
[Section 9.6.1 of the PGN spec](http://www.saremba.de/chessgml/standards/pgn/pgn-complete.htm#c9.6.1) defines how TimeControl has to be used. 

I should do the following:

* Find some games that use that TimeControl tag.
* Implement some rules to read them in a structured way.
* Define unit tests for all of them.

**Comment by mliebelt** [comment]
Found in the meanwhile the [spec handling time comments](https://github.com/mliebelt/pgn-spec-commented/blob/main/pgn-spec-supplement.md#4-time-handling-commands), and read it carefully. What I read is:

* There may be different time commands in the same comment, so the current structure does not work. Having one "clock" attribute is not sufficient therefore, and is replaced by multiple attributes named after the command. The following gives an example
```
 {[%cgt 01:00:10] [%egt 0:10:25]}
    ==> return json { cgt: "01:00:10", egt: "0:10:25" }
```
* Clock commands and other annotations may be mixed with normal comments, there is no rule that forbids that. So the following is legal: `1. e4 { Normal move [%cgt 00:04:53] Black's turn ...} e5`

I have already implemented the grammar (by doing #31 ) and will check the test cases and expand them if necessary.
